week1/homework/responses/1.1-bolt-intro.md

```diff
@@ -0,0 +1,11 @@
+# Introduction to Bolt JS
+
+Bolt JS is a framework, or a platform, for developing in JavaScript for Slack. Bolt **significantly** reduces the learning curve for Slack development by abstracting away some of the more complicated language barriers involved. In this issue, you will be familiarizing yourself with Bolt.
+
+## 📚 Reading Up
+
+Before getting into Bolt, [read up on Slack](https://api.slack.com/start/overview) and its use cases. This will help you get a better idea of what the projects that we will use Bolt to create. After you are done reading, you can start getting more into the technical details. Read [this article](https://api.slack.com/tools/bolt) on Bolt, then go through 2 or 3 code samples of your choice. The code samples can be found at the very bottom of the article.
+
+## ✍️ Reflection
+
+After you are done with both articles, write down three things you found interesting and would like to learn more about. Afterwards, write down three more things that you found difficult to understand.
```

Comment:
I like asking the students about what they found interesting/difficult to understand, but I think asking them a specific question about the content would "force" them to think more deeply about their understanding. At least for me as a student, I find when teachers ask me to write down interesting topics etc. I pick 3 random things and call it a day.

tl;dr **Ask students to present a use case of Slack they would be interested in learning more about and how a piece of code from the samples could apply to this idea. Afterwards, ask them to describe any issues they had with comprehending the content--ie. do you have any questions?**